crypto/asn1/a_int.c

```diff
@@ -289,7 +289,7 @@ static int asn1_get_int64(int64_t *pr, const unsigned char *b, size_t blen,
             ASN1err(ASN1_F_ASN1_GET_INT64, ASN1_R_TOO_SMALL);
             return 0;
         }
-        *pr = -(int64_t)r;
+        *pr = -(uint64_t)r;
```

Comment:
So at least in VS's log you get to see:

    crypto\asn1\x_long.c(79): warning C4146: unary minus operator applied to unsigned type, result still unsigned
    crypto\asn1\a_int.c(292): warning C4146: unary minus operator applied to unsigned type, result still unsigned
    crypto\bio\b_print.c(454): warning C4146: unary minus operator applied to unsigned type, result still unsigned